libs/client/ui/components/src/lib/back-button/back-button.tsx

```diff
@@ -4,10 +4,12 @@ import './back-button.scss';
 
 export interface BackButtonProps {
   defaultRouterLink: string;
+  invertColor?: boolean;
 }
 
 export const BackButton: React.FC<BackButtonProps> = ({
```

Comment:
Generell bezüglich deam BackButton:
Der backbutton brucht im `ion-header` immer an speziella spacing und des hosch du in jeder page dazua gmacht. Da des jetzt so oft vorkut also idee des ganze einfach in a globales `ion-header.scss` styling auslagern. Denn kannsch mine ganzen kommentare bezüglich verschachteln und so ignorieren.